### PR TITLE
Update pt-br.json

### PR DESCRIPTION
**Pull request by franklinbenini**
localization pt-br.json updated to 5.1.2

### CODE REVIEW

**Comment by narukaioh** [comment]
the portuguese it's fine, but I will review the changes in local

File: src/lang/pt-br.json [hunk 1]
```diff
@@ -9,103 +9,104 @@
...
-	"ARTIFACT.MIGHTY": "Poderoso",
-	"ARTIFACT.REGULAR": "Item normal",
-
-	"ATTACK.BLUNT": "Esmagamento",
-	"ATTACK.FEAR": "Medo",
-	"ATTACK.OTHER": "Outros",
-	"ATTACK.SLASH": "Cortante",
-	"ATTACK.STAB": "Golpear",
-
-	"ATTRIBUTE.AGILITY": "Agilidade",
```
Comment (aMediocreDad) [inline src/lang/pt-br.json:38]: The attributes should not be all caps. If they are the main tab on the character sheet will overflow and scroll as a result.

**Review by aMediocreDad — CHANGES_REQUESTED** [review]

File: src/lang/pt-br.json [hunk 2]
```diff
@@ -9,103 +9,104 @@
...
-	"ATTRIBUTE.EMPATHY": "Empatia",
-	"ATTRIBUTE.STRENGTH": "Força",
-	"ATTRIBUTE.WITS": "Astúcia",
```
Comment (aMediocreDad) [inline src/lang/pt-br.json:41]: The attributes should not be all caps. If they are the main tab on the character sheet will overflow and scroll as a result.

**Review by aMediocreDad — CHANGES_REQUESTED** [review]
Please fix the lines in the comment above.

File: src/lang/pt-br.json [hunk 3]
```diff
@@ -9,103 +9,104 @@
...
+  "ATTACK.BLUNT": "Esmagamento",
+  "ATTACK.FEAR": "Medo",
+  "ATTACK.OTHER": "Outros",
+  "ATTACK.SLASH": "Cortante",
+  "ATTACK.STAB": "Golpear",
+
+  "ATTRIBUTE.AGILITY": "AGILIDADE",
+  "ATTRIBUTE.EMPATHY": "EMPATIA",
+  "ATTRIBUTE.STRENGTH": "FORÇA",
+  "ATTRIBUTE.WITS": "ASTÚCIA",
```
Comment (aMediocreDad) [inline src/lang/pt-br.json:42]: The attributes should not be all caps. If they are the main tab on the character sheet will overflow and scroll as a result.

**Review by aMediocreDad — CHANGES_REQUESTED** [review]

**Review by franklinbenini — COMMENTED** [review]
fixed attributes caps

**Review by aMediocreDad — APPROVED** [review]
Thanks! I'll go ahead and merge this.

**Pull request merged**